app/src/main/java/com/infomaniak/mail/ui/main/thread/calendar/CalendarEventBannerView.kt

```diff
@@ -121,12 +157,19 @@ class CalendarEventBannerView @JvmOverloads constructor(
         }
     }
 
-    private fun setAttendees(attendees: RealmList<Attendee>) = with(binding) {
-        val iAmPartOfAttendees = attendees.any { it.isMe() }
-        notPartOfAttendeesWarning.isGone = iAmPartOfAttendees
-        participationButtons.isVisible = iAmPartOfAttendees && false // TODO : Display this when buttons click are implemented
+    private fun setAttendanceUi(attendees: List<Attendee>, shouldDisplayReplyOptions: Boolean) = with(binding) {
+        val userAsAttendee = attendees.findUser()
+        val iAmInvited = userAsAttendee != null
+
+        notPartOfAttendeesWarning.isGone = iAmInvited
```

Comment:
```suggestion
        val userIsInvited = attendees.findUser()
        notPartOfAttendeesWarning.isGone = userIsInvited != null
```

Reply:
Maybe we could also invert the assignement to 'isVisible = userIsInvited == null'